modules/changeSections.js

```diff
@@ -0,0 +1,39 @@
+const listBook = document.getElementById('listBooks');
+const addNewSection = document.getElementById('addNewBook');
+const contactSection = document.getElementById('contactSection');
+const sectionUno = document.getElementById('list');
+const sectionDos = document.getElementById('add-new');
+const sectionTres = document.getElementById('contact');
+
+const changeListBookClick = () => {
+  sectionUno.style.display = 'block';
+  sectionDos.style.display = 'none';
+  sectionTres.style.display = 'none';
+  listBook.style.color = 'blue';
+  addNewSection.style.color = 'black';
+  contactSection.style.color = 'black';
+};
+
+const changeAddNewSectionClick = () => {
+  sectionUno.style.display = 'none';
+  sectionDos.style.display = 'block';
+  sectionTres.style.display = 'none';
+  listBook.style.color = 'black';
+  addNewSection.style.color = 'blue';
+  contactSection.style.color = 'black';
+};
+
+const changeContactSectionClick = () => {
+  sectionUno.style.display = 'none';
+  sectionDos.style.display = 'none';
+  sectionTres.style.display = 'flex';
+  listBook.style.color = 'black';
+  addNewSection.style.color = 'black';
+  contactSection.style.color = 'blue';
+};
```

Comment:
[OPTIONAL] Well done overall 💯 but there seems to be some repetition happening here, we could try to us a helper function to make the code `DRY`. Here's a helper function that takes in the sections to display and an object that maps each link element to its respective color:
```
function changeSectionClick(sections, colors) {
  Object.entries(colors).forEach(([linkId, color]) => {
    const link = document.getElementById(linkId);
    link.addEventListener('click', () => {
      // hide all sections
      Object.values(sections).forEach(section => {
        section.style.display = 'none';
      });

      // show selected section
      sections[linkId].style.display = 'block';

      // set link colors
      Object.entries(colors).forEach(([id, color]) => {
        const link = document.getElementById(id);
        link.style.color = color;
      });
    });
  });
}
```
You can call this function 📞  by passing in the section elements and an object that maps 🗺️  each link element's id to its respective color:
```
changeSectionClick(
  { sectionUno, sectionDos, sectionTres },
  { listBook: 'blue', addNewSection: 'black', contactSection: 'black' }
);
```
This will simplify your code and make it more reusable if you need to add more sections and links in the future. 🤓